hx_lti_initializer/templates/ig/detail.html

```diff
@@ -53,9 +53,11 @@
 				context_id: "{{ course }}",
 				collection_id: "{{ collection }}",
 				object_id: {{ object }},
+				//Uhhh....this token is exposed -- TODO!!
 				token: "{{ token }}",
```

Comment:
@arthurian I tried taking the token from the frontend and running a curl request on the catch db and was able to dump the whole thing.

curl -i -X GET -H "Content-Type: application/json" -H "token-censored" "http://ec2-52-26-240-251.us-west-2.compute.amazonaws.com:8080/catch/annotator/search?limit=1000"